keyboards/crkbd/lib/layer_state_reader.c

```diff
@@ -3,11 +3,11 @@
 #include <stdio.h>
 #include "crkbd.h"
 
-#define L_BASE 0
-#define L_LOWER 8
-#define L_RAISE 16
-#define L_ADJUST 65536
-#define L_ADJUST_TRI 65560
+ #define L_BASE 0
+ #define L_LOWER (1<<_LOWER)
+ #define L_RAISE (1<<_RAISE)
+ #define L_ADJUST (1<<_ADJUST)
+ #define L_ADJUST_TRI (L_ADJUST|L_RAISE|L_LOWER)
```

Comment:
```suggestion
#define L_ADJUST_TRI (L_ADJUST|L_RAISE|L_LOWER)
```
Those spaces are necessary?

Reply:
Nope. Not sure how they slipped in!